lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -36,7 +37,7 @@ def scores(self):
         """
         List of all problem scores in the subsection.
         """
-        return list(self.locations_to_scores.itervalues())
+        return [score for score, _ in self.locations_to_scores.itervalues()]
```

Comment:
Consider renaming `self.locations_to_scores` to `self.locations_to_weighted_scores` since it's now a tuple of `(score, weight)`.
Also, need to update its docstring on L33 (`# dict of problem locations to their Score objects`).